src/components/input/styles.module.css

```diff
@@ -18,15 +18,12 @@
   border: 0;
   width: 100%;
   height: 4.8rem;
+  outline: none;
```

Comment:
oh, I think I might have missed this in the other PR.

this is a change you can do after you finish the challenge, if you have time, as this is not the focus of the challenge.
I'd suggest having all the styling of the input on the input element itself. You can have the wrapper div be used just to absolute position the icon. This way, you can use the css selectors to change the styling, instead of having to set the classes on JS.